platform.md

```diff
@@ -461,12 +461,22 @@ Usage:
       - All run-image config values SHALL be preserved unless this conflict with another requirement
     - MUST contain all buildpack-provided launch layers as determined by the [Buildpack Interface Specfication](buildpack.md)
     - MUST contain one or more app layers as determined by the [Buildpack Interface Specfication](buildpack.md)
-    - MUST contain a layer that includes `<launcher>`
+    - MUST contain a launcher layer containing:
+        - A file with the contents of the `<launcher>` file at path `/cnb/lifecycle/launcher`
+        - One symlink per buildpack provided `process` with name `/cnb/process/<type>` and target `/cnb/lifecycle/launcher`
     - MUST contain a layer that includes `<layers>/config/metadata.toml`
-    - MUST have `ENTRYPOINT=<launcher>`
+    - **If** `<process-type>` matches a buildpack-provided process:
+      - MUST have `ENTRYPOINT=/cnb/process/<process-type>`
+    - **If** `<process-type>` does not match a buildpack-provided process:
```

Comment:
Don't you prefer to write it as `if - else` instead of `if true` and then `if false`?

Reply:
For this particular one I think the multiple `If`s are clearer because the `Else` would be at the same level as many bullet points (not just the relevant `If`)